### PR TITLE
fix ssr example

### PR DESCRIPTION
**Pull request by DanielAmenou**
Fix server-side-rendering example using webpack-federation-module-id-plugin and webpack-federation-stats-plugin

This fix demonstrates how to use loadable-components with module federation, and how to avoid this issue https://github.com/gregberge/loadable-components/issues/640

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
Great work!!

**Review by ScriptedAlchemy — APPROVED** [review]

**Comment by todorpr** [comment]
@ScriptedAlchemy maybe I'm missing something, but the `server-side-rendering` example was working on my side before this fix. However something I feel is wrong even now is this:
```js
// server-side-rendering/website1/build/webpack.config.js/server.base.js

const remotePath = path.resolve(
    __dirname,
    "../../../website2/buildServer/container.js"
)
...

    new ModuleFederationPlugin({
      name: "website1",
      library: { type: "commonjs2" },
      filename: "container.js",
      remotes: {
        website2: remotePath // <- this line
```

In real life `website2` is supposed to be in a remote location, so don't we need here `remotePath` to be an URL instead of a relative path?

EDIT: Ok I just watched your video regarding the `multi-threaded-ssr` 
https://youtu.be/kOuoSBTCzl4?t=538 and realised why this relative path is and why it works. However I need a SSR solution where the remote is consumed from a URL.

File: server-side-rendering/website1/server/render.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { Helmet } from "react-helmet";
 import { ChunkExtractor } from "@loadable/server";
 import path from "path";
 import App from "../src/components/App";
-
+import { getMfChunks, createScriptTag, createStyleTag } from "./mfFunctions";
+import SomeComponent from "website2/SomeComponent";
```
Comment (debugpai) [inline server-side-rendering/website1/server/render.js:8]: Hey. I know this is a bit late to comment on this PR but I had a question about this import.

Is this mandatory to import all the remote modules you need in the application beforehand? Is there no way around this?

Comment (DanielAmenou) [reply server-side-rendering/website1/server/render.js:8]: Hi,
To my understanding, it is mandatory to import all the remote modules before rendering.
You can do it dynamically
https://webpack.js.org/concepts/module-federation/#dynamic-remote-containers

**Comment by philmoreira** [comment]
@ScriptedAlchemy just bumping @todorpr's note above. Will Module Federation ever be able to work server-side where the remote can be consumed from a URL (sans multi-threaded-ssr)?

File: server-side-rendering/website1/server/render.js (hunk 1 above)
Comment (IMalyugin) [reply server-side-rendering/website1/server/render.js:8]: Damn, that should actually be highlighted in the example, spent 2 days figuring out why example works, while my implementation does not. It's all cause of that "Unused import" that i totally ignored :)

**Pull request merged**